### DISCUSSION TITLE
Is RCP design a FTD or MTD? Not clear that it can be an MTD.

### DISCUSSION THREAD

**Discussion by userABCD-ship-it** [openthread/openthread#7882] [Q&A] [answered]
Hello,

I have worked through many of the examples.

If I want to configure a Linux device with either the siLabs or nRF RCP, is it possible to use this setup as a MED/SED? Or is the RCP configuration always an FTD?

It is not really clear from any of the documentation. There are choices for ot-cli-ftd & ot-cli-mtd, but nothing implies those will actually work with ot-ctl/ot-daemon. When using the Posix apps, everything implies the radio chipset should be flashed with ot-rcp, but it isn't clear the difference between ot-cli-ftd, ot-cli-mtd, & ot-rcp without dissecting the code manually.

Thoughts? Lastly, is there any documentation on what all of the options for ot-ctl are? I used 'help', but it is not easy finding the information on my own.

**Comment by jwhui** [comment, marked as answer]
When using an RCP, the Thread protocol primarily runs on a host processor that is separate from the RCP - see [Co-Processor Designs](https://openthread.io/platforms/co-processor).

`ot-cli-ftd` and `ot-cli-mtd` are used with SoC designs - see [Single-Chip, Thread-Only (SoC)](https://openthread.io/platforms#single-chip,-thread-only-soc).

OpenThread supports FTD and MTD configurations regardless of how your system is implemented, whether it is an SoC or Co-Processor design.

For CLI documentation, see [src/cli](https://github.com/openthread/openthread/tree/main/src/cli).
